app/matplotlib.ipynb

```diff
@@ -123,21 +131,25 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 6,
-   "outputs": [],
    "source": [
     "categories = [\"Category A\", \"Category B\", \"Category C\", \"Category D\"]\n",
     "sales = [350, 480, 210, 590]\n",
     "\n",
-    "# write your code here\n"
+    "plt.bar(categories, sales)\n",
+    "plt.title(\"Info about categories and sales\")\n",
+    "\n",
+    "plt.xlabel(\"Categories\")\n",
+    "plt.ylabel(\"Sales\")\n"
```

Comment:
You need to add `plt.xticks(rotation=45)` to rotate the x-ticks by 45 degrees, and `plt.show()` to display the bar plot.